.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: go
 go:
-- 1.6.2
+- 1.7.1
```

Comment:
Though we don't have to build agent and agent-plugins with same version of go, but to simplify I specify just 1.7 at https://github.com/mackerelio/mackerel-agent-plugins/pull/262/commits/4c06a5546a4034bf04dfb069d5cfe0425686151f.